package/kedro_viz/server.py

```diff
@@ -296,12 +296,11 @@ def _pretty_name(name: str) -> str:
     return " ".join(parts)
 
 
-def _pretty_modular_pipeline_name(modular_pipeline: str) -> str:
-    """Takes the namespace of a modular pipeline and prettifies the
-    last part to show as the modular pipeline name."""
-    chunks = modular_pipeline.split(".")
-    last_chunk = chunks[-1]
-    return _pretty_name(last_chunk)
+def _pretty_modular_pipeline_name(modular_pipeline_name: str) -> str:
```

Comment:
what's the rationale of taking it out of the pythonic way of thinking?

Reply:
I think if we want to keep the dots, there's not really a point in sending both the `id` which is basically just the full namespace as well as a sanitised `name`. WDYT?

Reply:
I'm not sure where I should comment on implementation. But using the analogy of a file path, I think may be helpful provide  the `absolute` path so that we can do stuff later without the backend as well the equivalent of  `pathlib.Path.name` i.e. the last part of the path.